docs/content/docs/3.server/3.cache.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Cache
+description: How to use cache storage with NuxtHub.
+---
+
+NuxtHub cache uses [Cloudflare Workers KV](https://developers.cloudflare.com/kv) to cache your server route responses or functions using Nitro's `cachedEventHandler` and `cachedFunction`.
+
+## Getting Started
+
+Enable the cache storage in your NuxtHub project by adding the `cache` property to the `hub` object in your `nuxt.config.ts` file.
+
+```ts [nuxt.config.ts]
+export default defineNuxtConfig({
+  hub: {
+    cache: true
+  }
+})
+```
+
+Once you deploy your project, you can access to the cache storage in the [NuxtHub admin](https://admin.hub.nuxt.com/). You can manage your cache entries in the `Cache` section of your project page.
+
+
+## Event handlers caching
+
+Using the `cachedEventHandler` function, you can cache the response of a server route. This function will cache the response of the server route into the NuxtHub cache storage.
+
+```ts [server/api/cached-route.ts]
+import type { H3Event } from 'h3'
+
+export default cachedEventHandler((event) => {
+  return {
+    success: true,
+    date: new Date().toISOString()
+  }
+}, {
+  maxAge: 60 * 60, // 1 hour
+  getKey: (event: H3Event) => event.node.req.url
+})
+```
+
+The above example will cache the response of the `/api/cached-route` route for 1 hour. The `getKey` function is used to generate the key for the cache entry.
```

Comment:
Could you add a `callout` or `note` and link to https://nitro.unjs.io/guide/cache#options ?